### PR TITLE
Formattering, tekst og eksempler

### PR DESCRIPTION
**Pull request by Mikkelinski**
## 📥 Proposed changes

Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.

## ☑️ Submission checklist

-   [ ] I have read the [CONTRIBUTING](https://github.com/fremtind/jokul/blob/master/CONTRIBUTING.md) doc
-   [ ] `yarn build` works locally with my changes
-   [ ] I have added tests that prove my fix is effective or that my feature works
-   [ ] I have added necessary documentation (if appropriate)

## 💬 Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did, what alternatives you considered, etc...

### CODE REVIEW

File: packages/text-input/README.md [hunk 1]
```diff
@@ -1,14 +1,13 @@
 # [`@fremtind/jkl-text-field`](https://fremtind.github.io/jokul/components/textfield/)
 
-## Om tekstfelt
+## Tekstfelt
+Tekstfelt lar brukerene legge inn data. Tekstfelt kan være korte, hvor brukeren kan oppgi en setning eller et navn, eller lange, der de kan skrive en lengre beskrivelse.
```
Comment (Steinop) [inline packages/text-input/README.md]: Har forsøkt å forenkle denne litt: "Tekstfelt lar brukerne legge inn egne opplysninger. Tekstfeltene kan være korte eller lange og gå over én eller flere linjer."

File: packages/text-input/README.md [hunk 2]
```diff
@@ -1,14 +1,13 @@
...
-### Validering og tegnbegrensning
+### Langt tekstfelt
+Dersom det er ønskelig at brukeren skriver en lengre tekst (eller trenger å ha mulighet til å kunne gjøre det) bør man bruke et langt tekstfelt. Dette feltet ekspanderer slik at det blir tydelig for brukeren at man kan skrive mye.
 
-Tekstfelt kan utsette en løsning for unødvendige feil eller behov for at fagpersoner sjekker dem. Det kan derfor være lurt å ha validering på feltet og oppgi hvor mange tegn brukerne kan legge inn.
-
-### Tekst
-
-Sørg alltid for å ha gode ledetekster og plassholdertekster. En hjelpetekst under ledeteksten kan også være nyttig.
+Når man bruker lange tekstfelt er det også viktig med en god ledetekst som forklarer hva som er ønsket input. Man kan også oppgi hvor mange tegn man tillater.
```
Comment (Steinop) [inline packages/text-input/README.md]: "Når man bruker lange tekstfelt er det viktig med en god ledetekst" 
(Dropp også)

File: packages/text-input/README.md (hunk 1 above)
Comment (Mikkelinski) [reply packages/text-input/README.md]: oppdatert

File: packages/text-input/README.md (hunk 2 above)
Comment (Mikkelinski) [reply packages/text-input/README.md]: oppdatert

**Pull request merged**